packages/contracts-bedrock/src/L2/L1Block.sol

```diff
@@ -14,6 +14,12 @@ contract L1Block is ISemver {
     /// @notice Address of the special depositor account.
     address public constant DEPOSITOR_ACCOUNT = 0xDeaDDEaDDeAdDeAdDEAdDEaddeAddEAdDEAd0001;
 
+    /// @notice Selector for 'NotDepositor()' error message.
+    uint32 public constant ERR_NOT_DEPOSITOR = 0x3cc50b45;
```

Comment:
Is there a way to use the custom error type and maybe there is a property on the error type that gives you the selector?

Reply:
We want the standard error abi tooling to just work with this